nengo_spa/actions.py

```diff
@@ -26,10 +30,15 @@ class Parser(object):
         'reinterpret': Reinterpret,
         'translate': Translate}
 
-    def __init__(self, vocabs=None):
-        if vocabs is None:
-            vocabs = {}
-        self.vocabs = vocabs
+    def __init__(self, locals_=None):
+        if locals_ is None:
```

Comment:
Could you add a comment about the function of `locals` here? Why does it default to `None`? How does it control iterating through frames?